common/src/main/java/com/wynntils/screens/overlays/OverlaySelectionScreen.java

```diff
@@ -24,9 +24,11 @@ public class OverlaySelectionScreen extends Screen {
     private static final int BUTTON_HEIGHT = 20;
 
     private OverlayList overlayList;
+    private final Screen lastScreen;
 
     public OverlaySelectionScreen() {
         super(new TranslatableComponent("screens.wynntils.overlaySelection.name"));
```

Comment:
I think in general there is a pattern in Minecraft where you send in the "previous" screen as the first argument to the Screen constructor. There used to be in 1.12 at least. I think we can try and follow it.

Reply:
I will see if there is such thing here. I don't think so though.